src/containerDeployment/dockerUtils.ts

```diff
@@ -0,0 +1,688 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import { exec } from "child_process";
+import { platform } from "os";
+import {
+    DockerCommandParams,
+    DockerStep,
+    DockerStepOrder,
+} from "../sharedInterfaces/containerDeploymentInterfaces";
+import { ApiStatus } from "../sharedInterfaces/webview";
+import {
+    connectionsArrayName,
+    defaultContainerName,
+    defaultContainerPort,
+    extensionName,
+    localhost,
+    localhostIP,
+    Platform,
+} from "../constants/constants";
+import { ContainerDeployment, msgYes } from "../constants/locConstants";
+import { TelemetryActions, TelemetryViews } from "../sharedInterfaces/telemetry";
+import { sendActionEvent } from "../telemetry/telemetry";
+import * as path from "path";
+import { FormItemOptions, FormItemValidationState } from "../sharedInterfaces/form";
+import { getErrorMessage } from "../utils/utils";
+import { Logger } from "../models/logger";
+
+/**
+ * The maximum length of the error text to display in the UI.
+ * If the error text exceeds this length, it will be truncated.
+ */
+const MAX_ERROR_TEXT_LENGTH = 300;

```

Comment:
Where is the number coming from? You should try to use CSS styling to truncate text with ellipsis. 

```
text-overflow: ellipsis;
```